### PR TITLE
feat: add script for updating config with precache starting block

### PR DESCRIPTION
**Pull request by nduchak**
Set starting block as `top_block_number - 50`

### CODE REVIEW

File: scripts/update-precache-blocks.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const SERVICES = {
+  rns: RNS_CONTRACTS,
+  storage: STORAGE_CONTRACTS,
+  triggers: TRIGGERS_CONTRACTS
+}
+
+const configFileName = `${NODE_ENV}.json5`
+const pathToConfigFile = path.resolve(__dirname, '../config', configFileName)
+
+const isExist = fs.existsSync(pathToConfigFile)
```
Comment (0xartem) [inline scripts/update-precache-blocks.js]: minor naming: `exists` or `doesExist`

File: scripts/update-precache-blocks.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
```
Comment (0xartem) [inline scripts/update-precache-blocks.js:1]: this script will be executed by the devops in their infra?

**Review by 0xartem — APPROVED** [review]
Good job

File: scripts/update-precache-blocks.js (hunk 2 above)
Comment (nduchak) [reply scripts/update-precache-blocks.js:1]: Yes, probably it will be executed inside the cache docker container

Comment (nduchak) [reply scripts/update-precache-blocks.js:1]: Now, I am thinking about defining another strategy for deployment from scratch. There we should define starting blocks as blocks on which contract was deployed

**Review by 0xartem — APPROVED** [review]
nice, so basically you query the explorer API for the deployment tx depending on the network. Let's see how we can integrate it on the devops side, the CD part will be interesting.

**Pull request merged**